app/models/visitor.rb

```diff
@@ -2,9 +2,12 @@ class Visitor < ActiveRecord::Base
   include Person
   extend FreshnessCalculations
 
+  attr_accessor :validate_contact_list_matching
```

Comment:
This will go. Based on a discussion with @StupidCodeFactory I'm going to try to get rid of this flag altogether from the model.